app/src/main/java/gis2018/udacity/pomodoro/CountDownTimerService.java

```diff
@@ -82,7 +82,6 @@ public int onStartCommand(Intent intent, int flags, int startId) {
                 .setColor(getResources().getColor(R.color.colorPrimary))
                 .setContentIntent(pendingIntent)
                 .setContentText("Countdown timer is running")
-                .addAction(R.drawable.complete, "Complete", completeActionPendingIntent)
```

Comment:
Hey @somenath1435 ,
Removing this line would remove the Complete action button altogether, even when the Session timer is running.

What is required is for it to be removed only when the break is running.